src/test/scala/chiselTests/BoringUtilsSpec.scala

```diff
@@ -376,4 +376,65 @@ class BoringUtilsSpec extends ChiselFlatSpec with ChiselRunners with Utils with
       "propassign a, bar.a_bore"
     )()
   }
+
+  behavior.of("BoringUtils.drive")
+
+  it should "fail on probes" in {
+    class Foo extends RawModule {
+      val a = Wire(Bool())
+      val p = ProbeValue(a)
+    }
+
+    class Bar extends RawModule {
+      val foo = Module(new Foo)
+
+      BoringUtils.drive(foo.p) := 1.B
+    }
+
+    val e = the[Exception] thrownBy circt.stage.ChiselStage.emitCHIRRTL(new Bar)
+
+    e.getMessage should include("requirement failed: cannot drive a probe from BoringUtils.drive")
+  }
+
+  it should "bore ports for driving hardware" in {
+    class Foo extends RawModule {
+      val a = Wire(Bool())
+    }
+
+    class Bar extends RawModule {
+      val foo = Module(new Foo)
+
+      BoringUtils.drive(foo.a) := 1.B
+    }
+
+    val chirrtl = circt.stage.ChiselStage.emitCHIRRTL(new Bar, Array("--full-stacktrace"))
```

Comment:
Nit: `--full-stacktrace` is unnecessary. Ditto below.

Reply:
Oops yes leftover from debugging and then copied. Thanks.